addons/mass_mailing/views/assets.xml

```diff
@@ -23,7 +23,7 @@
                 box-sizing: border-box !important;
             }
             .o_layout :not(.fa) {
-                font-family: Arial, sans-serif !important;
+                font-family: Arial,Helvetica Neue,Helvetica,sans-serif;
```

Comment:
Is this line really relevant to this particular commit ? I don't get it.

Reply:
We add the option to change the font family and this is now the default font family so I changed it in the general css as well.